src/utils/get-organized-arguments.ts

```diff
@@ -287,53 +291,72 @@ export default async function getOrganizedArguments(): Promise<OrganizedArgument
 
 		// Validate data, if necessary
 		if (mergedSpec.data.accepts && mergedSpec.data.accepts instanceof Array) {
+			// Initialize holder of acceptable values
+			const acceptables: string[] = [];
+
 			// Find unrecognized data by removing acceptable values
+			//  -> Space-padding to guard against partial matches
+			//  -> Acceptables list should be presented in longest to shortest ordering if multi-word phrases exist
 			let remnants = ` ${organizedArguments.data} `;
 
 			for (const current of mergedSpec.data.accepts) {
-				remnants = remnants.replace(` ${current} `, ' '); // Space-padding to guard against partial matches
+				if (remnants.indexOf(` ${current} `) >= 0) {
+					acceptables.push(`${current}`);
+
+					remnants = remnants.replace(` ${current} `, ' ');
+				}
 			}
 
 			remnants = remnants.trim();
 
-			// Error if unknown remnants remain
+			// Error if remnants remain
```

Comment:
Why was this comment changed?

Reply:
I wanted to avoid people seeing 'unknown' and confusing this with 'undefined' or similar, which would address a different issue entirely.

Reply:
I'm not sure that's really compelling enough of a reason; let's keep this comment unchanged.